### ISSUE TITLE
(feature request) Segmented dropdown button

### ISSUE THREAD

**Issue by zanozik** [Dogfalo/materialize#4491]
Is there any plans on developing **[segmented dropdown buttons](https://material.io/guidelines/components/buttons.html#buttons-dropdown-buttons)**? 
These are really useful when there is a list (or a table) and each item (or row) has several actions. This could be done easily by segmenting a regular `data-activates='dropdown'` button, making dropdown appear only when second segment is pressed (covering the whole segmented button).

**Comment by fega** [comment]
Since this project is an implementation of the material design guidelines, I believe that it will be added at any moment.

that about contributing with a PR?

**Comment by zanozik** [comment]
Yes, I know the connection to Google Material, hence the question. I am not sure if I am qualified to make a PR with a new feature like this (unlike, say, a bug patch), since many components will have to be changed, which means by the time this PR is approved it may become conflicting. Moreover, I don't know how devs would like to go about making this segmented dropdown button, as there are tons of ways. 
What I can do, though, is to showcase it using existing css components, tweaking them a little bit, if that would do any good to the cause :)

**Comment by zanozik** [comment]
Just a small [codepen showcase]( https://codepen.io/itdesigns/pen/YZbNqa/)
